src/service/print/docxHelper.js

```diff
@@ -394,6 +297,36 @@ async function renderMCQ(question, questionCounter, marks) {
   return data;
 }
 
+function setOptions(questionOptions, questionOpt, imageProperties, printLanguage, i) {
```

Comment:
Hope you enjoyed writing the code.. looks fine.

@vaivk369 @shubhambansaltarento

Reply:
yes i did, Thank you for guidance.